compiler/src/scala/quoted/internal/impl/ExprImpl.scala

```diff
@@ -1,6 +1,5 @@
-package dotty.tools.dotc.quoted
-
-import scala.quoted._
+package scala.quoted
+package internal.impl
```

Comment:
What is the consideration here for prefering `scala.quoted. internal.impl` over `dotty.tools.dotc.quoted`?

Reply:
We need to add `ExprImpl` and `TypeImpl` in the `scala` package as `Expr`/`Type` have `private[scala]` constructors. If we have that it also makes sense to put the other related classes in there for consistency.